### PR TITLE
Make simplecov return the correct coverage

### PR DESCRIPTION
**Pull request by beatrichartz**
Currently the coverage for this repo is reported as 44% - good news, Hanami actually has 93% coverage 🎉

### CODE REVIEW

File: test/support/coverage.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+module Coverage
```
Comment (davydovanton) [inline test/support/coverage.rb]: What do you think, may be is better to use class instead module? :)

**Comment by davydovanton** [comment]
good job, I liked it 👍

File: test/support/coverage.rb (hunk 1 above)
Comment (beatrichartz) [inline test/support/coverage.rb]: I quite like the `module` - `module_function` approach - I can change it to a class with class functions if that suits the style of the repo better, do you want me to?

Comment (davydovanton) [inline test/support/coverage.rb]: no, it was only question. For me class is more clear than `module_function`, that's why I asked.

Comment (beatrichartz) [inline test/support/coverage.rb]: I don't mind changing it either way :) Just let me know. Thanks for taking the time to review this!

**Comment by jodosha** [comment]
@beatrichartz Wonderful! I was looking at this problem today. Probably the coverage is even higher, but there are some tests that aren't counted because ran in a separated process. See `script/ci`.

Do you think it's possible to merge the coverage for multiple processes? See https://github.com/colszowka/simplecov/issues/147#issuecomment-6603303

**Comment by beatrichartz** [comment]
@jodosha good catch, added the integration tests in, simplecov merges them automatically. Also made jRuby run in debug mode so the correct coverage can be provided from the Travis build.

File: test/support/coverage.rb (hunk 1 above)
Comment (joneslee85) [inline test/support/coverage.rb]: TIL `module_function`, IMO I quite like it

**Comment by jodosha** [comment]
@beatrichartz Thank you! Just merged this PR! 🎉

**Pull request merged**